### ISSUE TITLE
Animate Image to editor

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-addons#10]
Borderline MVP - if one interaction could be made beautiful ...

When I user taps to share/edit an image, it would be great to clone the image and nicely animate it's appearance in the edit space.

**Comment by xmatthewx** [comment]
depends on #19

**Comment by xmatthewx** [comment]
low priority

**Comment by toolness** [comment]
We implemented this in the [embedded example](http://xmatthewx.github.io/webmaker-addons/embedded/) we made for the heartbeat before last, but implementing this in the add-on will be trickier because the sidebar and the user's current tab don't actually exist on the same web page.

Two possible solutions:
- Dave Camp knows about a special overlay area above web content that developer tools use to overlay information about the page. Presumably that can also overlay the sidebar and thus be used for the transition effect.
- We might be able to "hack" the animation without resorting to Firefox internal APIs by doing it in a similar way that one drags a window from one monitor to another on a multi-monitor desktop setup: the annoying thing is that even if we do everything right, the image will still get broken by the border between the sidebar and the web content (just like on a multi-monitor setup, the window is "broken" by the physical space between the two monitors).

**Comment by xmatthewx** [comment]
I wouldn't worry about this too much. If we could fade in on the sidebar that would be good for now. Just enough transition so it feels like something meaningful happened.

**Comment by toolness** [comment]
Alright, in 37e273c043d44bdfb7aed21a3d61253de4cd7eb1 I actually did the second option I described in my previous comment, and it actually looks quite nice, but it requires two-way communication between the web content and the sidebar, which I don't have enough time to implement for the demo tomorrow. You can see this currently at the [embedded demo](http://xmatthewx.github.io/webmaker-addons/embedded/) if you check the inscrutable "Animate transition from both sides" checkbox.

Anyways, I ended up using the same plumbing to do something a bit less awesome in a93abd110a5e6f1776d56079b8a3e2e5b5dca7d8, but I think it's decent enough for the demo tomorrow:

![blah](https://cloud.githubusercontent.com/assets/124687/6179704/9ebee2f6-b2e9-11e4-9a27-5b0e681c6f76.gif)

**Comment by davidascher** [comment]
nice, definitely helps convey the spirit of the thing.
